### ISSUE TITLE
False syntax errors since latest update

### ISSUE THREAD

**Issue by FPtje** [wmertens/sublime-nix#5]
Today I noticed that valid code is highlighted as syntactically incorrect. I reckon it has to do with the latest commit.

Example:
```nix
{ config, lib, pkgs, ... }:

with lib;

{
  config = {
    systemd.services.example = with pkgs; {
      description = "asdf";
      serviceConfig = {
        ExecStart = "asdf";
      };
    };
  };
}
```

Result: 

![2017-06-02-094117_354x219_scrot](https://cloud.githubusercontent.com/assets/1202014/26715908/b4a510c0-4777-11e7-83c6-9aa4254e5897.png)

The code is syntactically valid. The pink highlights are incorrect.

**Comment by wmertens** [comment]
thank you for the heads-up, hopefully v2.3.1 restores normalcy.

**Comment by FPtje** [comment]
It hasn't. The latest version on package control still has this issue.

**Comment by wmertens** [comment]
@FPtje Strange, the code you gave renders correctly in github, and they use https://github.com/github/linguist which presumably uses the latest sublime-nix. Can you confirm that your installed sublime package is v2.3.1 and it has the changes included in https://github.com/wmertens/sublime-nix/commit/5fa227a7c42cd3ecd4dd86721f1a8d5dc460df8a ?

**Comment by FPtje** [comment]
It is version 2.3.1 and it does contain the changes included in that revision. I checked by opening `nix.YAML-tmLanguage` inside the `nix.sublime-package` and observing line 332 being as follows:

```yaml
    begin: (?![\w'-])(with|assert)(?![\w'-])
```

**Comment by wmertens** [comment]
Ok, thanks for checking. I will investigate and fix soon.

**Comment by demin-dmitriy** [comment]
@FPtje until fixed version is not available as a package in package control repository you can try master branch version, e.g. on linux this should do the trick:
```
$ git clone https://github.com/wmertens/sublime-nix ~/.config/sublime-text-3/Packages/Nix
```

**Comment by FPtje** [comment]
I had already reverted to a previous version that didn't have the bug

**Comment by wmertens** [comment]
In the meantime, I hope the latest release fixed the bug?

**Comment by FPtje** [comment]
Looks like it's fixed on ebf06571f3cc2b30a7f88bbd4eb371b72ce828c2. Thanks!
